src/Backend/Modules/Location/Engine/Model.php

```diff
@@ -79,11 +87,16 @@ public static function get(int $id): array
      */
     public static function getAll(): array
     {
-        return (array) BackendModel::getContainer()->get('database')->getRecords(
-            'SELECT i.*
-             FROM location AS i
-             WHERE i.language = ? AND i.show_overview = ?',
-            [BL::getWorkingLanguage(), true]
+        /** @var EntityRepository $repository */
+        $repository = BackendModel::get('doctrine.orm.default_entity_manager')->getRepository(Location::class);
+
+        $locations = $repository->findAll();
```

Comment:
Gaat dit geen issue geven als er geen locations zijn?

Reply:
No, it will just return an empty array

Reply:
Nope, it always returns an array so the array map will just be skipped if it's empty